New Bank 2FA Method 946

There appears to be a new tan mode with the number 946, it is not yet configurable in the drop down menu.
My hourly cron import failed, starting yesterday at UTC 2025-07-31T20:00:01Z

`Fatal error: Uncaught AssertionError: Your bank did not accept your tan mode 944. Accepted modes are: 946 in /app/FinTsFactory.php:63 Stack trace: #0 /app/FinTsFactory.php(63): assert(false, 'Your bank did n...') #1 /app/FinTsFactory.php(35): App\FinTsFactory::get_tan_mode(Object(Fhp\FinTs), Object(Symfony\Component\HttpFoundation\Session\Session)) #2 /app/Choose2FADevice.php(25): App\FinTsFactory::create_from_session(Object(Symfony\Component\HttpFoundation\Session\Session)) #3 /app/index.php(55): App\StepFunction\Choose2FADevice() #4 {main} thrown in /app/FinTsFactory.php on line 63`

--- a/app/public/html/collecting-data.twig
+++ b/app/public/html/collecting-data.twig
@@ -60,6 +60,7 @@
                     <option value="940">DKB-App (940)</option>
                     <option value="942">mobileTAN (942)</option>
                     <option value="944">SecureGoTAN (944)</option>
+                    <option value="946">SecureGo plus (946)</option>
                     <option value="962">Smart-TAN plus manuell (962)</option>
                     <option value="972">Smart-TAN plus optisch (972)</option>
                     <option value="982">Smart-TAN photo (982)</option>

